msprime/demography.py

```diff
@@ -744,10 +745,12 @@ def _events_html(self, events, title=None):
         data = []
         for event in events:
             class_name = event.__class__.__name__
+            camel_case = re.sub(r"(?<!^)(?=[A-Z])", "_", class_name).lower()
+            add_method = f"msprime.Demography.add_{camel_case}"
             # TODO change this to stable when 1.0 is released.
             type_html = (
-                "<a href='https://tskit.dev/msprime/docs/latest/api.html#msprime."
-                f"{class_name}'>{event._type_str}</a>"
+                "<a href='https://tskit.dev/msprime/docs/latest/api.html#"
```

Comment:
Should this normally be stable? Do we change this after the pre-releases are over?

Reply:
Yep, agreed. That's what the TODO just above is supposed to say :wink:

Reply:
Haha, bad idea to review on mobile!!